qml/main.qml

```diff
@@ -40,6 +40,143 @@ ApplicationWindow {
 
     cover: Qt.resolvedUrl("pages/CoverPage.qml")
 
+    ListModel {
+        id: regions
+
+        ListElement {
+            name: QT_TR_NOOP("Finland")
```

Comment:
This seems not to work
```
[W] unknown:119 - file:///usr/share/harbour-jollaopas/qml/pages/SettingsPage.qml:119:52: Unable to assign [undefined] to QString
[W] unknown:46 - file:///usr/share/harbour-jollaopas/qml/js/reittiopas.js:46: TypeError: Cannot read property 'boundarycirclelat' of undefined
```
and I think that this option is bad user experience for address search even if it works. This option is also very confusing for the end user and I would left this out entirely.

Reply:
Sorry, it was missing the identifier field. I changed the name to ”Whole Finland” and moved it to the end of the list to discourage selecting it a bit.
The address search experience is, of course, suboptimal (or at least requires a bit of skill in adding the city searched for). My original idea was that it covers any regions that are not available as separate regions, but in the latest commit I changed the finland option to not limit transport modes – so basically it works just the same as the opas.matka.fi search. I personally would find this useful for checking train timetables etc.